tests/test_reckoner.py

```diff
@@ -271,7 +272,8 @@ def test_minimum_version(self):
         self.assertRaises(MinimumVersionException, self.c._compare_required_versions)
 
     def test_plot_course(self):
-        self.configure_subprocess_mock('', '', 0)  # Lots more work do do here with the installation of the list of charts
+        self.configure_subprocess_mock('', '',
+                                       0)  # TODO: Lots more work do do here with the installation of the list of charts
```

Comment:
Is this formatting intentional?

Reply:
probably my IDE trying make PEP8 standards.  I can put it back if you want

Reply:
Line length maybe? My pep8 formatter doesn't automatically fix that error. I don't really care, was just curious if it was an